pandas/tseries/tests/test_period.py

```diff
@@ -101,15 +101,21 @@ def test_timestamp_tz_arg(self):
                          pytz.timezone('Europe/Brussels').normalize(p).tzinfo)
 
     def test_timestamp_tz_arg_dateutil(self):
-        import dateutil
+        if sys.platform != 'win32':
+            from dateutil.tz import gettz
+        else
```

Comment:
all of the tests should import ``tslib._dateutil_gettz`` rather than computing it.